### ISSUE TITLE
Call to browser->fake() fails with ScriptTimeoutException of 1 second

### ISSUE THREAD

**Issue by flap152** [NoelDeMartin/laravel-dusk-mocking#12]
Laravel 5.7

I have tests that sometimes fail when using `$browser->fake()` with ScriptTimeoutException "script did not return in 1 seconds" (or words to that effect).

I traced it to this Browser method that explicitly sets the timeout to 1 second juste before executing the script. Setting to 5 seconds fixes the problem.

```
    public function executeJavascriptRequest($method, $url, $params = [], $useSession = true) {
        $this->driver->manage()->timeouts()->setScriptTimeout(1);
        $result = $this->driver->executeAsyncScript(
```

I can't extend + override the method because it uses private methods.
I'd like to avoid forking your project just for this, but especially: Why just 1 second? I would agree the default is too long at 30 seconds, but even a local http request could take time if the test machine becomes busy... like PHPStorm re-indexes ;-)

**Comment by NoelDeMartin** [comment]
I think I was using the timeout because it was taking too long to fail in some tests, and I wanted to have a faster feedback loop. But yes, 1 second is an arbitrary number, no particular reason to use that.

Although 5 seconds may also be an arbitrary number, so what I'll do is leave the default as 1 second and implement a way to configure this, so that you can set it to whatever you like :).

Thanks for reporting!

**Comment by flap152** [comment]
Works for me. And done darn quick too... Many thanks! 
    On Friday, June 7, 2019, 1:54:26 p.m. GMT-4, Noel De Martin <notifications@github.com> wrote:  
 
 
Closed #12 via d1286f4.
